docs/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "fluentui-docs",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next",
```

Comment:
What is the `next` command?

Reply:
it's a low-config react framework to serve up the mdx pages, simpler than gatsby.  It's present so the repo can serve docs reasonable for local dev (and sample code can be build-checked via jsx in mdx), mdx is navigable and readable on github, and polished publish by the gatsby fluent-site